tests/test_converter.py

```diff
@@ -812,16 +806,13 @@ def test_qasm_from_file(self, tmpdir, recorder):
         assert recorder.queue[5].parameters == []
         assert recorder.queue[5].wires == Wires([3])
 
-        assert len(record) == 11
+        assert len(record) == 5
```

Comment:
As far as I recall that the `CU1Gate`  was added at the time because it was the only gate that lacked the `to_matrix` method. Hence it couldn't be converted.

Reply:
Nice! So it turns out the test was failing because the converter now _works_ automatically for the CU1 gate 😆